gwpy/table/gravityspy.py

```diff
@@ -194,8 +193,8 @@ def search(cls, gravityspy_id, howmany=10,
         """
         from astropy.utils.data import get_readable_fileobj
         import json
-        from six.moves.urllib.error import HTTPError
-        from six.moves import urllib
+        import urllib
+        from urllib.error import HTTPError
```

Comment:
Can these `import`s be moved to the module leve, with `urllib` replaced by the full `from urllib.parse import urlencode`?

Reply:
@duncanmmacleod, I agree with the suggestion, and I took a stab at this with the most recent commit.